public/resources/core/js/api/party/individual.service.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @author Francisco de la Vega <fdelavega@conwet.com>
+ *         Jaime Pajuelo <jpajuelo@conwet.com>
+ *         Aitor Magán <amagan@conwet.com>
+ */
+
+
+(function () {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .factory('Individual', IndividualService);
+
+    function IndividualService($q, $resource, URLS, User) {
+        var Individual = $resource(URLS.PARTY_MANAGEMENT + '/individual/:partyId', {}, {
+            update: {method: 'PUT'}
+        });
+
+        Individual.prototype.getUser = getUser;
+
+        var MEDIUM_TYPES = {
+            EMAIL_ADDRESS: 'email address',
+            TELEPHONE_NUMBER: 'telephone number',
+            POSTAL_ADDRESS: 'postal address'
+        };
+
+        return {
+            MEDIUM_TYPES: MEDIUM_TYPES,
+            create: create,
+            detail: detail,
+            update: update,
+            launch: launch
+        };
+
+        function create(data) {
+            var deferred = $q.defer();
+
+            Individual.save(data, function (individualCreated) {
+                deferred.resolve(individualCreated);
+            }, function (response) {
+                deferred.reject(response);
+            });
+
+            return deferred.promise;
+        }
+
+        function detail() {
```

Comment:
In some cases you would be interested in retrieving information of different users (not necessarily the one logged in).